change/@fluentui-babel-preset-global-context-be7bd550-e553-4a5f-9aa9-50399a752c06.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
Does it make sense to opt for the change type `none` rather than `prerelease/patch` for packages, considering the PR changes do not impact the source code?

Reply:
I think the idea behind using `prerelease/patch` is to trigger a release and ship a new version of these packages that doesn't contain `package.json#scripts` anymore

Reply:
@TristanWatanabe explained it pretty well - so yeah that's the reasoning here. ty!

overall I'm ok to keep those changes as `none` and ship it with real code change in future.

do you have any preference here/can we keep as is ? @dmytrokirpa @TristanWatanabe 

**extra context:** there will be also additional package.json cleanup in future ( removing devDependencies completely to not leak additional metadata to public ) - so similar approach in terms of "change files" would be taken.

Reply:
I'm okay to keep it as is, but I think such a release will not bring a lot of value for package consumers.

Reply:
fortunately that's true (our packages didn't expose any postinstall hooks etc )

but it's a change that needs reflection via version bump to follow semver.